graph/GRAPH.h

```diff
@@ -0,0 +1,117 @@
+#include<bits/stdc++.h>
```

Comment:
I thought [the contribution guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#code) explicitly said don't use `bits/stdc++.h` ? Check out [this Stack Overflow answer](https://stackoverflow.com/questions/31816095/why-should-i-not-include-bits-stdc-h)